### PR TITLE
fix: check if artist exists in AuctionResultsByFollowedArtists resolver

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses a bug discussed [here](https://artsy.slack.com/archives/C02BAQ5K7/p1624982653469000).

### CODE REVIEW

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Thank you so much for immediately following up on this!

File: src/schema/v2/me/auction_results_by_followed_artists.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ const AuctionResultsByFollowedArtists: GraphQLFieldConfig<
       const { body: followedArtists } = await followedArtistsLoader(gravityArgs)
 
       const followedArtistIds = followedArtists.map(
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/schema/v2/me/auction_results_by_followed_artists.ts:76]: Great idea! 👍

[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/schema/v2/me/auction_results_by_followed_artists.ts:76]: Thanks, please feel free to merge once you commit then. I will assign you to this PR

**Comment by olerichter00** [comment]
#squashongreen

**Pull request merged**